Client/mods/deathmatch/logic/CFileReader.h

```diff
@@ -0,0 +1,36 @@
+#ifndef CFILEREADER_H
```

Comment:
File is missing the copyright/etc. comment at the top.
Also, use **#pragma once**